wrappers/java/ci/android.test.sh

```diff
@@ -30,14 +30,12 @@ build_test_artifacts(){
         SET_OF_TESTS=''
 
         # This is needed to get the correct message if test are not built. Next call will just reuse old results and parse the response.
-        RUSTFLAGS="-L${TOOLCHAIN_DIR}/sysroot/usr/${TOOLCHAIN_SYSROOT_LIB} -L${LIBZMQ_LIB_DIR} -L${SODIUM_LIB_DIR} -L${INDY_LIB_DIR} -L${OPENSSL_DIR} -lsodium -lzmq -lc++_shared -lindy" \
-        LIBINDY_DIR=${INDY_LIB_DIR} \
+        RUSTFLAGS="-L${TOOLCHAIN_DIR}/sysroot/usr/${TOOLCHAIN_SYSROOT_LIB} -L${LIBZMQ_LIB_DIR} -L${SODIUM_LIB_DIR} -L${OPENSSL_DIR} -lsodium -lzmq -lc++_shared" \
             cargo test ${BUILD_TYPE} --target=${TRIPLET} ${SET_OF_TESTS} --no-run
 
         # Collect items to execute tests, uses resulting files from previous step
         EXE_ARRAY=($(
-            RUSTFLAGS="-L${TOOLCHAIN_DIR}/sysroot/usr/${TOOLCHAIN_SYSROOT_LIB} -lc -lz -L${LIBZMQ_LIB_DIR} -L${SODIUM_LIB_DIR} -L${INDY_LIB_DIR} -lsodium -lzmq -lc++_shared -lindy" \
-            LIBINDY_DIR=${INDY_LIB_DIR} \
+            RUSTFLAGS="-L${TOOLCHAIN_DIR}/sysroot/usr/${TOOLCHAIN_SYSROOT_LIB} -lc -lz -L${LIBZMQ_LIB_DIR} -L${SODIUM_LIB_DIR} -L${INDY_LIB_DIR} -lsodium -lzmq -lc++_shared" \
```

Comment:
This argument should also be removed: `.. -L${INDY_LIB_DIR} ...`

Reply:
We don't link libindy.{a|so|dylib} in any way. All code from libindy.a is already part of libvcx.so

`nm -C libvcx.so | egrep ' T (indy|vcx)_' | wc -l `=> 328

Reply:
Yes, I am just pointing out the argument is not necessary but still present